From 75baa1751973378cb96fb204b0a18a74e5caa2d1 Mon Sep 17 00:00:00 2001
From: Rouven Czerwinski <r.czerwinski@pengutronix.de>
Date: Wed, 17 Feb 2021 14:03:20 +0100
Subject: [PATCH] serialdriver: remove pyserial version check

This check isn't required on NixOS, since pyserial within NixOS already
contains the patches.

Signed-off-by: Rouven Czerwinski <r.czerwinski@pengutronix.de>
---
 labgrid/driver/serialdriver.py | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/labgrid/driver/serialdriver.py b/labgrid/driver/serialdriver.py
index 126f674e..59a92269 100644
--- a/labgrid/driver/serialdriver.py
+++ b/labgrid/driver/serialdriver.py
@@ -27,12 +27,6 @@ class SerialDriver(ConsoleExpectMixin, Driver, ConsoleProtocol):
         bindings = {"port": "SerialPort", }
     else:
         bindings = {"port": {"SerialPort", "NetworkSerialPort"}, }
-    if version.parse(serial.__version__) != version.Version('3.4.0.1'):
-        message = ("The installed pyserial version does not contain important RFC2217 fixes.\n"
-                   "You can install the labgrid fork via:\n"
-                   "pip uninstall pyserial\n"
-                   "pip install https://github.com/labgrid-project/pyserial/archive/v3.4.0.1.zip#egg=pyserial\n")  # pylint: disable=line-too-long
-        warnings.warn(message)
 
     txdelay = attr.ib(default=0.0, validator=attr.validators.instance_of(float))
     timeout = attr.ib(default=3.0, validator=attr.validators.instance_of(float))
-- 
2.30.0

